templates/java/org/yarp/Loader.java.erb

```diff
@@ -90,6 +94,23 @@ public class Loader {
         return string;
     }
 
+    private String[] loadStringsSequence() {
+        int count = loadVarInt();
+
+        assert count >= 0;
+
+        String[] strings = new String[count];
+        Charset charset = Charset.forName("US-ASCII");
```

Comment:
There is `StandardCharsets.US_ASCII` to avoid the extra lookup.
This probably deserves a comment like `// all warning and error messages only contain ASCII characters`